### ISSUE TITLE
Pulsatile mass in extremity

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10832]
**Preferred term label:**
Pulsatile mass in extremity

**Synonyms**

**Definition (free text, please give PubMed ID)**
A noticeable lump or swelling in the arm or leg that you can feel with you fingers and this lump is beating or pulsating, similar to the feeling of a heartbeat

**Parent term (use hpo.jax.org/app)**
This is similar to https://hpo.jax.org/browse/term/HP:6000039 and our pending request for "Pulsatile abdominal mass" but with different anatomical localization.

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Arteriovenous malformation, extremity
Ulnar artery thrombosis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001159
- label: Pulsatile mass in extremity
- parents: [Abnormality of limbs (HP:0040064)](https://hpo.jax.org/app/browse/term/HP:0040064)
- pmid: 
- def: A noticeable lump or swelling in the arm or leg that you can feel with you fingers and this lump is beating or pulsating, similar to the feeling of a heartbeat.
- comment: This finding may be observed with conditions such as arteriovenous malformation located in the arm or leg or withulnar artery thrombosis.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10832
- synonyms:
